addons/website_hr_recruitment/static/tests/tours/website_hr_recruitment.js

```diff
@@ -133,4 +133,29 @@
     },
 ]);
 
+    // This tour addresses an issue that occurred in a website form containing
+    // the 'hide-change-model' attribute. Specifically, when a model-required
+    // field is selected, the alert message should not display an undefined
+    // action name.
+    wTourUtils.registerWebsitePreviewTour('model_required_field_should_have_action_name', {
+        test: true,
+        url: '/jobs',
+    }, () => [{
+        content: "Select Job",
+        trigger: "iframe h3:contains('Guru')",
+    }, {
+        content: "Apply",
+        trigger: "iframe a:contains('Apply')",
+    },
+    ...wTourUtils.clickOnEditAndWaitEditMode(),
+    {
+        content: "click on the your name field",
+        trigger: "iframe #hr_recruitment_form div.s_website_form_model_required",
```

Comment:
@visp-odoo In this version, I think `iframe` becomes `:iframe` in tour selectors.

Reply:
Done changes :+1: